tawazi/__init__.py

```diff
@@ -7,6 +7,6 @@
 from .config import cfg
 from .consts import Resource
 
-__version__ = "0.3.2"
+__version__ = "0.4.0.dev0"
```

Comment:
hmm, I am unsure as to keep dev versions on the main branch tbh

Reply:
maybe we should do a rc ?

Reply:
this is just a place holder... we shall change it later of course according to what best suits us